docs/config.md

```diff
@@ -0,0 +1,135 @@
+# Config file options
+
+SOCI has a multitude of variables that can customize SOCI behavior to best suit users' needs. This doc will break down the possible values for each variable, where the variable can be found, and what it can do.
+
+This doc assumes the usage of a TOML file. Per TOML specs, Anything in [brackets] is the required header for the set of variables.
+
+An example configuration file can be found in config/config.toml.
+
+The following format will be used throughout this doc:
+
+## File path
+### StructName [toml_header]
+- `toml_var_name` — StructVarName (type). Brief description. Defaults to some value.
+
+#
+
+# Uncategorized
```

Comment:
Not really, it's just here because there's no TOML variable corresponding to each of these variables despite residing inside their respective structs.